examples/article.amp.html

```diff
@@ -144,11 +144,26 @@
       background-color: #f4f4f4;
     }
   </style>
+  <script async custom-element="amp-analytics" src="../dist/v0/amp-analytics-0.1.max.js"></script>
+  <script async custom-element="amp-user-notification" src="../dist/v0/amp-user-notification-0.1.max.js"></script>
+  <script async src="/examples/viewer-integr.js"></script>
   <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
   <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
-  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async src="../dist/amp.js"></script>
```

Comment:
Please revert.

There is a build process that puts these files into examples.build with the URLs pointing to local. We want the examples to be copy-pastable.